extension/content.css

```diff
@@ -399,21 +403,50 @@ svg.octicon.octicon-star.dashboard-event-icon {
 }
 
 .reaction-summary-item div,
+.reaction-summary-item div a,
 .reaction-summary-item div img {
 	display: inline-block;
 }
 
+.reaction-summary-item div a {
+	transition: margin-left 0.2s;
```

Comment:
@bfred-it - Sorry, can we add `margin-left: -2px` here too to make a perfect match with the `:hover` version? (`4px` apart in both "normal" and the `:hover` version).

I know, I have a problem. 🙈